ably/types/message.py

```diff
@@ -201,29 +202,22 @@ def from_encoded(obj, cipher=None):
         )
 
     @staticmethod
-    def __update_empty_fields(proto_msg: dict, msg: dict, msg_index: int):
-        if msg.get("id") is None or msg.get("id") == '':
-            msg['id'] = f"{proto_msg.get('id')}:{msg_index}"
-        if msg.get("connectionId") is None or msg.get("connectionId") == '':
-            msg['connectionId'] = proto_msg.get('connectionId')
-        if msg.get("timestamp") is None or msg.get("timestamp") == 0:
-            msg['timestamp'] = proto_msg.get('timestamp')
+    def __update_empty_fields(proto_msg: dict, msg_list: List[dict]):
+        for msg_index, msg in enumerate(msg_list):
```

Comment:
Btw, response from `enumerate` tend to return type of `msg_index` and `msg` set to `any` in `pycharm`. Is it the same in your IDE too?
That's why I was using a simple loop to iterate, where typings are available